.github/workflows/build.yml

```diff
@@ -22,6 +22,15 @@ jobs:
         make format
         git diff --exit-code
 
+    - name: Install ZMQ
+      run: |
+        wget https://github.com/jedisct1/libsodium/releases/download/1.0.19-RELEASE/libsodium-1.0.19.tar.gz
```

Comment:
hmm doesn't seem like `make test` worked on CI. Can you fix that?

Reply:
will do